2023/did-wg.html

```diff
@@ -151,12 +150,10 @@ <h2>Motivation and Background</h2>
 level changes on the Recommendation.
         </p>
         <p>
-The Working Group will also seek consensus around DID Method specifications, and
-may produce related documents.
+The Working Group will also take over the work started by the Credenials Community Group around <a href="https://w3c-ccg.github.io/did-resolution/">DID Resolution</a>, to define common requirements, algorithms including their inputs and results, architectural options, and various considerations for the DID resolution and DID URL dereferencing processes.
         </p>
         <p>
-The Working Group will begin working toward a specification for DID Resolution
-and DID URL Dereferencing.
+It is expected that these two goals will be taken by distinct task forces in the WG.
```

Comment:
I would prefer to not have 2x meetings... hopefully thats not what is being implied by this sentence.

Reply:
The charter does not say that.

Reply:
@OR13 To be honest, I would also prefer to not have 2x meetings :wink: but as @iherman points out, the charter makes no assumption either way. Charters usually don't go into the detail of regular group meetings, this is up to the chairs to decide what's the best meeting frequency for each task-force is.

Are you ok with resolving this conversation?

Reply:
seeing no protest, I resolve this conversation